source/synthDrivers/oneCore.py

```diff
@@ -135,6 +163,9 @@ class OneCoreSynthDriver(SynthDriver):
 	}
 	supportedNotifications = {synthIndexReached, synthDoneSpeaking}
 
+	_ocSpeechToken: Optional[ctypes.POINTER]
+	_queuedSpeech: List[Union[str, Tuple[Callable[[ctypes.POINTER, float], None], float]]]

```

Comment:
Shouldn't this typing be in the `init` method? Doesn't having it here indicate these are expected to be used as class variables?

Reply:
Fixed in e630724